src/main/resources/model/bibframe_authority.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "bibframe-authority",
+  "eventBodyJavaClass": "org.folio.search.domain.dto.BibframeAuthority",
```

```suggestion
  "eventBodyJavaClass": "org.folio.search.domain.dto.BibframeAuthority",
  "reindexSupported": true
```
Should be defined to not break snapshot builds.